locales/en/common.json

```diff
@@ -0,0 +1,23 @@
+{
+  "intro-title": "Technology that makes your life easier",
+  "button-continue": "continue",
+  "button-contact-us": "contact us",
+  "WelcomeTo React": "Bienvenue à React et react-i18next",
```

Comment:
Is this French? Is that correct? In appears in the other files too.

Reply:
I have to correct that

Reply:
fixed in commit https://github.com/capcatit/capcatit.github.io/pull/14/commits/0c434f527307558c2056b8e649f286d23177144d